Finished my task

mono-tmt opened the pull request:
## Create CSS Animation

mono-tmt committed:
[add] kenji header.ejs

t

mono-tmt committed:
[add] add kenji.scss

t

mono-tmt committed:
[update] finished animation on desktop

t

mono-tmt committed:
[update] Add CSS to mobile-menu on sp

t

mono-tmt committed:
:+1: Add JavaScript

t

mono-tmt committed:
[update]  display menu-list when clicked a btn

t

mono-tmt committed:
[update] undisplay menu-list when clicked anywhere

t

mono-ito commented on src/ejs/pages/kenji.ejs:
```diff
@@ -14,11 +14,73 @@
+    <!-- <div class="nav-trigger"></div> -->
+    <header class="l-header">
+        <div class="l-header__inner">
+            <div class="l-header__logo">
+                <span>Hello</span>
+                <span>World</span>
+            </div>
```
不要なタグは削除してください。

mono-tmt replied:
taiouzumi-

mono-ito commented on src/ejs/pages/kenji.ejs:
```diff
@@ -14,11 +14,73 @@
+            </button>
+        </div>
+    </header>
+    <nav class="c-mobile-menu">
+        <ul class="c-mobile-menu__main">
+            <li class="c-mobile-menu__item">
+                <a class="c-mobile-menu__link" href="#">Menu01</a>
+            </li>
+            <li class="c-mobile-menu__item">
+                <a class="c-mobile-menu__link" href="#">Menu02</a>
+            </li>
+            <li class="c-mobile-menu__item">
+                <a class="c-mobile-menu__link" href="#">Menu03</a>
+            </li>
+            <li class="c-mobile-menu__item">
+                <a class="c-mobile-menu__link" href="#">Menu04</a>
+            </li>
+        </ul>
+    </nav>
```
PC版とスマホ版でコードを分けていますが、
分けずに一つにまとめてください。

mono-tmt replied:
taiouzumi-

mono-ito replied:
これがまだ対応されていません。

mono-tmt replied:
> これがまだ対応されていません。 

対応済み( 主観的に一つにまとめたつもりです... )

> PC版とスマホ版でコードを分けていますが、分けずに一つにまとめてください。

ご指摘箇所の認識をボクがズレて解釈してるかもしれないです。

mono-ito replied:
具体的に言うと、以下の35行目～50行目のコードの削除をお願いします。
```
    <nav class="c-mobile-menu">
        <ul class="c-mobile-menu__main">
            <li class="c-mobile-menu__item">
                <a class="c-mobile-menu__link" href="#">Menu01</a>
            </li>
            <li class="c-mobile-menu__item">
                <a class="c-mobile-menu__link" href="#">Menu02</a>
            </li>
            <li class="c-mobile-menu__item">
                <a class="c-mobile-menu__link" href="#">Menu03</a>
            </li>
            <li class="c-mobile-menu__item">
                <a class="c-mobile-menu__link" href="#">Menu04</a>
            </li>
        </ul>
    </nav>
```

20～27行目のコードにPC用のスタイルがあたっていますが、
スマホ用のスタイルもあててください。
```
            <nav class="l-header__nav">
                <ul class="l-header__list">
                    <li class="l-header__item"><a class="l-header__link" href="#"><span>Menu01</span></a></li>
                    <li class="l-header__item"><a class="l-header__link" href="#"><span>Menu02</span></a></li>
                    <li class="l-header__item"><a class="l-header__link" href="#"><span>Menu03</span></a></li>
                    <li class="l-header__item"><a class="l-header__link" href="#"><span>Menu04</span></a></li>
                </ul>
            </nav>
```

mono-tmt replied:
## 修正箇所
- 35行目～50行目のコードの削除(HTML)
→ 対応済み

- 20～27行目のコード(HTML)にスタイルを当てる
→対応済み

mono-ito replied:
OKです。

mono-ito commented on src/ejs/pages/kenji.ejs:
```diff
@@ -14,11 +14,73 @@
+                    <li class="l-header__item"><a class="l-header__link" href="#">Menu04</a></li>
+                </ul>
+            </nav>
+            <button class="c-mobile-menu__btn">
```
ここが「c-」なのは若干違和感があるので、「l-header~~~」で良いと思います。

mono-tmt replied:
done

mono-ito replied:
OKです

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,275 @@
 // [環境設定]
-@import '../../foundation/_variables';
-@import '../../foundation/_mixin';
+@import "../../foundation/_variables";
```
コードの前に、デザイン再現上の指摘。

・ナビゲーションは横幅に対してリキッドデザインでお願いします。
・ナビゲーションの背景色がデザインと違います。
・ボーダーの色がデザインと違います。
・三角アイコンの位置がデザインと違います。
・ホバー時の緑の下線の位置がデザインと違います。

mono-tmt replied:
done

mono-ito replied:
・ナビゲーションは横幅に対してリキッドデザインでお願いします。
　→ナビゲーションの右に不要な余白があります。

・ナビゲーションの背景色がデザインと違います。
　→OK

・ボーダーの色がデザインと違います。
　→OK

・三角アイコンの位置がデザインと違います。
　→三角アイコンが無くなっています。

・ホバー時の緑の下線の位置がデザインと違います。
　→緑線の高さが違います。

[以下追加指摘]
・ホバーした後に緑線にカーソルをのせると、ホバーがはずれます。
・縦のグレーの枠線が太いです。

mono-tmt replied:
プッシュ済み

mono-ito replied:
・三角アイコンの位置がデザインと違います。
　→三角アイコンが無くなっています。

・ホバー時の緑の下線の位置がデザインと違います。
　→緑線の位置が違います。
　　緑線の左右とグレー枠線の間は8pxというデザイン意図です。

・ホバーした後に緑線にカーソルをのせると、ホバーがはずれます。
　→OK

・縦のグレーの枠線が太いです。
　→OK

[以下追加指摘]
4つめのボタンのグレー枠線の右側は不要です。

mono-tmt replied:
#4 修正箇所
**三角アイコンが無くなっています。**
→ 対応済み ( HTML の追加 )

**緑線の左右とグレー枠線の間は8pxというデザイン意図です。**
→ 対応済み

**4つめのボタンのグレー枠線の右側は不要です。**
→ 対応済み

mono-tmt replied:
## 修正箇所
SP版のボタン、クリック時のボタンのアニメーションをデザインの指示通りに
→ 対応済み

SP版のボタン、クリック時のメニューリストのアニメーションをデザインの指示通りに
→ 対応済み

mono-ito replied:
OKです

mono-tmt committed:
[fix] fixing two things that pointed out
- 不要なタグの削除
- class名の変更とそれに伴う修正
[変更前] c-mobile-menu__btn [変更後] l-header__menu-btn

t

mono-tmt committed:
[fix] fix a thing that pointed out
- リキッドデザインに修正
- ナビーゲーションの背景色を修正
- ボーダーの色を修正
- 三角アイコンの一の修正
- ホバー時の緑の下線の位置がデザインを修正

t

mono-tmt committed:
[fix] fixing a thing that pointed out
PC版とスマホ版のコードを一緒にする

t

mono-tmt committed:
[fix] PC版でモバイルメニューが表示iされる問題を解決

t

mono-tmt committed:
[fix] fixing the five thing that pointed out
- ナビゲーションの右に不要な余白があります。[修正]
- 三角アイコンが無くなっています。[追加]
- 緑線の高さが違います。[修正]
- ホバーした後に緑線にカーソルをのせると、ホバーがはずれます。[修正]
- 縦のグレーの枠線が太いです。[修正]

確認お願いします！

t

mono-tmt committed:
[add] change the color when hovered btn

t

mono-tmt committed:
[add] 追加漏れ

t

mono-tmt committed:
[fix] fixing three things that pointed out
- 三角アイコンが無くなっています [修正] (HTMLの追加漏れ)
- 緑線の左右とグレー枠線の間は8pxというデザイン意図です。[修正]
- 4つめのボタンのグレー枠線の右側は不要です。[修正]

ご確認よろしくお願いします！

t

mono-tmt committed:
[fix] Update animation when clicked menu-btn

t

mono-tmt committed:
[fix] fixing a thing that pointed out

t

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,224 @@
+    & > a {
+      color: $cFont;
+    }
+    &:first-child {
+      border-top: 1px solid $cBorder;
+    }
```
PCレイアウトの一つ目だけボーダートップ無しになっていますが、
ボーダートップは全て無しでおｋ

mono-tmt replied:
## 修正箇所
ボーダートップは全て無しでおｋ
→ 対応済み

SP版のメニューリストをデザインの指定通りに近づけました！

確認お願いします

mono-ito replied:
OKです

mono-tmt committed:
[fix] add border-top

t

mono-tmt committed:
[fix] remove border-top

t

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,230 @@
+    @include mq-up() {
+      position: static;
+      display: block;
+      height: 100%;
+      width: 100%;
+      margin: 0 auto;
```
不要かも

mono-tmt replied:
## 修正箇所
- 不要なプロパティの削除
→ 対応済み

mono-tmt committed:
[fix] remove it that dont need

t

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+      }
+    }
+    @include mq-up() {
+      position: relative;
+      text-align: center;
```
不要っぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+          }
+        }
+      }
+      span {
+        position: relative;
+        &::before {
+          position: absolute;
+          top: 0;
+          left: -26px;
+          content: "";
+          margin-top: 7px;
+          border: 7px solid transparent;
+          border-top-width: 5px;
+          border-bottom-width: 5px;
+          border-left-color: $cBlack;
+          transition: all 0.6s;
+        }
```
109行目以降と重複してるっぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+      border-top: 1px solid $cBorder;
+    }
+    @include mq-up() {
+      padding: 0;
```
不要っぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+    @include mq-up() {
+      padding: 0;
+      background-color: $cBg;
+      width: calc(100% / 4);
```
素直に25%で良い気がする

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+      padding: 0;
+      background-color: $cBg;
+      width: calc(100% / 4);
+      position: relative;
```
不要っぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+  }
+
+  &__list {
+    padding: 0;
```
不要っぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100px;
```
不要っぽい

mono-ito commented on src/styles/assets/css/kenji.scss:
```diff
@@ -1,9 +1,209 @@
+  left: 0;
+  height: 100px;
+  width: 100%;
+  background-color: transparent;
```
不要っぽい
